### PR TITLE
Add SUSE CaaS Platform and openSUSE Kubic to adopters

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Adding both products to the adopters docs.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
PTAL @sysrich @kkaempf
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -3,3 +3,10 @@
 Below is a non-exhaustive list of CRI-O adopters in production environments:
 
 * [Red Hat's Openshift Container Platform](https://www.openshift.com/) uses CRI-O as the only supported CRI implementation starting with the release of OpenShift 4. CRI-O was chosen because it provides a lean, stable, simple and complete container runtime that moves in lock-step with Kubernetes, while also simplifying the support and configuration of clusters.
+* [SUSE CaaS Platform](https://www.suse.com/products/caas-platform) uses CRI-O
+  since version 3. It has been initially supported as technology preview in
+  version 3 and moved to the default Kubernetes container runtime since version
+  4 as replacement for Docker.
```
Comment (TomSweeneyRedHat) [inline ADOPTERS.md:9]: nit
```suggestion
  4 as a replacement for Docker.
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -3,3 +3,10 @@
...
+* [openSUSE Kubic](https://kubic.opensuse.org) is a Certified Kubernetes
+  distribution based on openSUSE MicroOS. It uses CRI-O as supported container
```
Comment (TomSweeneyRedHat) [inline ADOPTERS.md:11]: nit
```suggestion
  distribution based on openSUSE MicroOS. It uses CRI-O as a supported container
```

Comment (saschagrunert) [reply ADOPTERS.md]: Fixed :+1:

File: ADOPTERS.md (hunk 1 above)
Comment (saschagrunert) [reply ADOPTERS.md]: Fixed :+1:

**Review by mrunalp — APPROVED** [review]

**Comment by saschagrunert** [comment]
@sysrich @kkaempf is the added content fine from your side?

**Comment by haircommander** [comment]
you'll need a rebase @saschagrunert

**Comment by saschagrunert** [comment]
/hold cancel
/retest

:heavy_check_mark: Rebased

**Pull request merged**